library/src/test/java/bio/terra/pfb/utils/CompareOutputUtils.java

```diff
@@ -22,9 +22,7 @@ public class CompareOutputUtils {
   public static void assertJavaPfbIsPyPFB(
       String fileName, PfbCommandType commandType, String filePath, FileExtension fileExtension)
       throws IOException {
-    String pythonOutput;
-    pythonOutput =
-        Files.readString(Paths.get(getPyPfbOutputFilePath(fileName, commandType, fileExtension)));
```

Comment:
was a specific reason this was done this way originally?

Reply:
ha wait, spoke too soon and now see that you just moved this into a separate function :) ignore